common/src/com/elikill58/negativity/common/protocols/AimBot.java

```diff
@@ -46,6 +46,69 @@ public AimBot() {
 		super(CheatKeys.AIM_BOT, CheatCategory.COMBAT, Materials.TNT, CheatDescription.VERIF);
 	}
 
+	// this check seems stupid, although it's a must-have
```

Comment:
Why those types of comment, not directly about how it works but only not productive ?